python/setup.py

```diff
@@ -342,7 +342,7 @@ def run(self):
 
     def get_proton_cmake_args(self):
         cmake_args = get_thirdparty_packages([get_json_package_info(), get_pybind11_package_info()])
-        cupti_include_dir = get_env_with_keys(["CUPTI_INCLUDE_PATH"])
+        cupti_include_dir = get_env_with_keys(["CUPTI_INCLUDE_PATH", "TRITON_CUPTI_PATH"])
```

Comment:
it seems redundant, why can't we use `CUPTI_INCLUDE_PATH` or replace it?

Reply:
Yeah, I think `CUPTI_INCLUDE_PATH` can be used

Reply:
The problem is probably that `TRITON_CUPTI_PATH` was used in `download_and_copy`. I think replacing `CUPTI_INCLUDE_PATH ` with `TRITON_CUPTI_PATH`/include would also work and might be more consistent.

Reply:
Yes, you're correct. By setting `TRITON_CUPTI_PATH`, cupti package won't be downloaded and thus the default `CUPTI_INCLUDE_PATH` at `triton/third_party` won't find anything and will cause a build time error.

From our side, we can easily set `export CUPTI_INCLUDE_PATH=${TRITON_CUPTI_PATH}` in CI build to bypass this, but I still think it would be convenient and consistent to look into both paths given the current setup.py configuration.

Reply:
Can you try to replace `CUPTI_INCLUDE_PATH` with something like `TRITON_CUPTI_PATH/include`? We don't need two env variables with the same purpose.

Reply:
It seems like `TRITON_CUPTI_PATH` already ends with `/include` so we probably don't need the `include` there? https://github.com/triton-lang/triton/blob/54960ca07a0596917007d907895ca9c2e4ec6fd9/python/setup.py#L479-L486

I did a quick search for `CUPTI_INCLUDE_PATH` in the repo https://github.com/search?q=repo%3Atriton-lang%2Ftriton%20CUPTI_INCLUDE_PATH&type=code and seems like that's the only place it existed in the repo. It was added here https://github.com/triton-lang/triton/pull/3756 .

I don't mind replacing `CUPTI_INCLUDE_PATH` with `TRITON_CUPTI_PATH` in `get_proton_cmake_args`, as long as it doesn't break others' CI setup. Should I try it?